### PR TITLE
fix: TairClient not implement TairCmdable(ExZAddManyScore, ExZAddMany…

### PR DESCRIPTION
**Pull request by exuan**
rt

### CODE REVIEW

**Comment by yangbodong22011** [comment]
@exuan thanks for your contribution.

File: tair/tair.go [hunk 1]
```diff
@@ -2,9 +2,12 @@ package tair
 
 import (
 	"context"
+
 	"github.com/go-redis/redis/v8"
 )
 
+var _ TairCmdable = (*TairClient)(nil)
```
Comment (yangbodong22011) [inline tair/tair.go:9]: why we need this line?

File: tair/taircommands.go [hunk 2]
```diff
@@ -12,13 +13,13 @@ type tairCmdable func(ctx context.Context, cmd redis.Cmder) error
 type TairCmdable interface {
 	// TairZset
 	exZAdd(ctx context.Context, key string, p *ExZAddArgs, member ...ExZAddMember) *redis.IntCmd
-	ExZAddManyScore(ctx context.Context, key string, member string, scores ...float64) *redis.IntCmd
```
Comment (yangbodong22011) [inline tair/taircommands.go]: float is right api，we need change `joinScoresToString` function to float.
refer [here](https://github.com/alibaba/alibabacloud-tairjedis-sdk/blob/master/src/main/java/com/aliyun/tair/tairzset/LeaderBoard.java#L66)

File: tair/tair.go (hunk 1 above)
Comment (exuan) [reply tair/tair.go:9]:
```
This is a relatively common Go idiom equivalent to Java's "implements", Rust's "impl ... for", etc.
It

asserts that types implement expected set of interfaces (declaration of intent), guarding against breakage caused by method signature changes
improves readability by making things explicit & acting as an additional form of documentation
helps with code navigation/generation (e.g. in GoLand Alt+Insert on (*Type)(nil) gives Implement Missing Methods option).
```
https://github.com/uber-go/guide/issues/25

Comment (yangbodong22011) [reply tair/tair.go:9]: thanks, got it.

File: tair/taircommands.go (hunk 2 above)
Comment (exuan) [reply tair/taircommands.go]: pr it

**Review by yangbodong22011 — APPROVED** [review]

**Comment by yangbodong22011** [comment]
released v1.1.4(bring a breaking change)
https://github.com/alibaba/tair-go/releases/tag/v1.1.4

**Pull request merged**